netbox/extras/forms.py

```diff
@@ -71,7 +71,7 @@ def get_custom_fields_for_model(content_type, filterable_only=False, bulk_edit=F
                     default_choice = cf.choices.get(value=initial).pk
                 except ObjectDoesNotExist:
                     pass
-            field = forms.TypedChoiceField(
+            field = CSVCustomFieldChoiceField(
```

Comment:
This is a general-purpose function for retrieving custom fields. There should be nothing specifically related to CSV import here.

Reply:
Okay, I've named it `CustomFieldChoiceField` and did some minor cleanup on that class.